.github/workflows/image-push.yml

```diff
@@ -0,0 +1,82 @@
+name: image-push
+
+on:
+  push:
+    branches:
+      - master
+      - stable
+    tags:
+      - '[0-9]+.*'
```

Comment:
The [pattern-matching available](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags) isn't quite as robust as in Dockerhub, so this is a bit more generic than the existing config.